e2e/e2e-scenario.sh

```diff
@@ -168,6 +169,7 @@ kubectl apply -f pod-nginx.yaml
 waitForPodStartTime=$(date +%s)
 for i in $(seq 1 10); do
     set +e
+    kubectl get pods -o wide | grep $podName
```

Comment:
workaround to make e2e pass because of pods not coming up?

Reply:
to show the pod status if it's not already in running